hr_holidays_imposed_days/models/hr_imposed_holidays.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class HrHolidaysImposed(models.Model):
+    _name = 'hr.holidays.imposed'
+    _description = 'Manage imposed holidays'
+
+    name = fields.Char(required=True)
+    date = fields.Date(required=True)
```

Comment:
a nice addition could be to have a date start and date end to be able to impose a week of holiday (lots of company in France for instance impose the week of Aug. 15 to their employees, or the week between christmas and new year).